.github/workflows/docker.yml

```diff
@@ -0,0 +1,18 @@
+name: Test docker image
+on: pull_request
+env:
+  DOCKER_BUILDKIT: 1
+  DATABASE_URL: sqlite:///
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update pip
+      run: pip install setuptools --upgrade
```

Comment:
Can you explain why this has to be added?

Reply:
That was an attempt to fix the `bdist_wheel error` I am having